### DISCUSSION TITLE
Center of Buoyancy

### DISCUSSION THREAD

**Discussion by kasia129** [capytaine/capytaine#119] [Q&A] [answered]
Is it possible to get the center of buoyancy/volume from Capytaine? I can imagine it must be calculated at least internally at some point for the purpose of hydrostatic matrix computation?

**Comment by mancellin** [comment, marked as answer]
Hydrostatics are not implemented in Capytaine at the moment, so there is no way to compute the center of buoyancy. The recommended way is to use Meshmagick (see e.g. #64).

**Reply by kasia129** [reply]
Thank you!
